### ISSUE TITLE
ExecutionException: java.lang.NoClassDefFoundError: org/apache/commons/codec/digest/DigestUtils

### ISSUE THREAD

**Issue by uckelman** [vassalengine/vassal#6809]
| Bugzilla |  |
|----------|----------|
| ID | BZ9643 |
| Reported | 2012-12-15 08:56:00 +0100 |
| Modified | 2021-05-02 22:14:32 +0200 |
| Product | VASSAL |
| Component | External |
| Version | 3.2.2 |
| Platform | All |
| OS | All |
| Status | TRIAGED |
| Resolution | None |
| Priority | unspecified |
| Severity | normal |
| Milestone | --- |
| Reporter | [Joel Uckelman](mailto:uckelman@nomic.net) |
| Assignee | [Bugs](mailto:bugs@vassalengine.org) |
| CC | b.easton@exemail.com.au |

| Attachments | |
|-------|-------|
| [errorLog](http://www.vassalengine.org/files/bz/8865) | [Joel Uckelman](mailto:uckelman@nomic.net) 2012-12-15 08:56:00 +0100 text/plain 4625 the errorLog |

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2012-12-15 08:56:31 +0100
<pre>---- Reported by uckelman@nomic.net  ----

Reported by troy@troysterling.com

Opening 'Sword and Sail' module form Mac Finder

--- Bug imported by uckelman@nomic.net 2012-12-15 08:56 MST  ---

This bug was previously known as _bug_  at http://www.vassalengine.org/trackershow_bug.cgi?id=
Imported an attachment (id=8865)

The original submitter of attachment 8865 is unknown.
   Reassigning to the person who moved it here: uckelman@nomic.net.</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2012-12-16 04:49:47 +0100
<pre>*** Bug 9635 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2012-12-16 04:56:45 +0100
<pre>I can't reproduce this. Emailed user for details.</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2012-12-16 13:15:11 +0100
<pre>User reported this happened only once.</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2013-02-07 13:50:01 +0100
<pre>*** Bug 9748 has been marked as a duplicate of this bug. ***</pre>

#### [Brent Easton](mailto:b.easton@exemail.com.au) 2013-06-02 04:53:29 +0200
<pre>*** Bug 10216 has been marked as a duplicate of this bug. ***</pre>

#### [Brent Easton](mailto:b.easton@exemail.com.au) 2013-10-02 23:12:39 +0200
<pre>*** Bug 10478 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2013-12-22 15:11:24 +0100
<pre>*** Bug 10670 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2013-12-22 15:11:38 +0100
<pre>*** Bug 10650 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2014-05-27 23:06:11 +0200
<pre>*** Bug 10941 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2014-05-27 23:06:22 +0200
<pre>*** Bug 10808 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2014-05-27 23:06:32 +0200
<pre>*** Bug 10839 has been marked as a duplicate of this bug. ***</pre>

#### [Brent Easton](mailto:b.easton@exemail.com.au) 2014-06-02 05:11:56 +0200
<pre>*** Bug 10874 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2014-08-19 12:04:50 +0200
<pre>*** Bug 11334 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2014-09-16 23:09:53 +0200
<pre>*** Bug 11410 has been marked as a duplicate of this bug. ***</pre>

#### [Brent Easton](mailto:b.easton@exemail.com.au) 2016-10-08 13:53:23 +0200
<pre>*** Bug 12273 has been marked as a duplicate of this bug. ***</pre>

#### [Brent Easton](mailto:b.easton@exemail.com.au) 2016-10-08 13:53:27 +0200
<pre>*** Bug 12056 has been marked as a duplicate of this bug. ***</pre>

#### [Brent Easton](mailto:b.easton@exemail.com.au) 2016-10-08 13:53:32 +0200
<pre>*** Bug 11553 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2020-07-18 17:37:54 +0200
<pre>*** Bug 13160 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2020-07-18 17:39:05 +0200
<pre>*** Bug 12905 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2020-07-18 17:40:05 +0200
<pre>*** Bug 12849 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2020-07-18 17:41:35 +0200
<pre>*** Bug 12606 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2020-07-18 17:41:52 +0200
<pre>Also reported by:

dragan321@comcast.net
kennethcrist1@gmail.com
pglowacki49@gmail.com</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2020-07-18 17:55:13 +0200
<pre>Also reported by:

radar726@yahoo.com</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2020-10-27 17:01:37 +0100
<pre>*** Bug 13261 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2020-10-27 17:01:54 +0100
<pre>Reported against 3.3.2 by darrylmyers01@gmail.com.</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2021-04-08 01:36:35 +0200
<pre>Amazingly this still happens in 3.5.5!</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2021-04-08 01:36:40 +0200
<pre>*** Bug 14349 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2021-05-02 16:29:47 +0200
<pre>*** Bug 14426 has been marked as a duplicate of this bug. ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2021-05-02 22:14:32 +0200
<pre>*** Bug 14445 has been marked as a duplicate of this bug. ***</pre>

**Comment by uckelman** [comment]
Huh, we haven't had one of these with 3.6 yet. (Not that I can come up with any reason why we should have had them before...)
